packages/web/server/queries/complex/route-token-out-given-in.ts

```diff
@@ -100,7 +102,13 @@ export async function getRouter(
           }
 
           if (pool["@type"] === COSMWASM_POOL_TYPE) {
-            return new TransmuterPool(pool as CosmwasmPoolRaw);
+            pool = pool as CosmwasmPoolRaw;
+
+            if (AstroportPoolCodeIds.includes(pool.code_id)) {
+              return new AstroportPclPool(pool);
```

Comment:
Let's pass in the base node query URL here instead of hardcoding in pools package